src/pretalx/cfp/templates/cfp/event/user_submission_edit.html

```diff
@@ -142,11 +142,11 @@ <h3>{% trans "Withdraw submission" %}</h3>
             </div>
         </div>
     {% elif submission.state == "accepted" or submission.state == "confirmed" %}
-        <h3>{% trans "Cancel submission" %}</h3>
+        <h3 id="change_info">{% trans "Cancel submission" %} / {% trans "change slot count" %}</h3>
```

Comment:
I don't think adding this information here is all that helpful. Speakers will have to contact organisers for serious changes regardless, and grouping slot count and deletion is unintuitive.